docs/user/editor/stops.md

```diff
@@ -2,37 +2,66 @@
+To begin editing stops, click the map marker icon button on the lefthand navigation bar (outlined in red).
 
-![screenshot](../../img/edit-stops.png)
+![edit-stops](https://datatools-builds.s3.amazonaws.com/docs/stops/edit-stops.png)
 
-## Selecting a stop
+### Selecting a stop
 
 Choose a stop from the list or search by stop name in the dropdown.
 
 You can also **zoom into the map** while the stop list is visible and once you're close enough you'll begin to see stops displayed. Click one to begin editing its details.
 
-## Creating a stop: right-click on map
+### Creating a stop
 
-To create a new stop, **right-click on the map** in the location you would like to place the stop. **Note:** as with all newly created items (except patterns), the new stop will not be saved until the save icon (💾) is clicked.
+To create a new stop, **right-click on the map** in the location you would like to place the stop. 
 
-## Moving a stop
+**Note:** as with all newly created items (except patterns), the new stop will not be saved until the save icon (💾) is clicked.
 
-To move a selected stop simply **click and drag the stop to the new location**. Or, if already you know the latitude and longitude coordinates, you can copy these into the text fields. After moving the stop, click save to keep the changes.
+### Editing a stop
+Once a stop is created or selected, the following parameters are required:
+- **Stop ID (`stop_id`):** Identifies a stop, station, or station entrance.
+- **Location (`stop_lat/stop_lon`):** These are defined by latitude and longitude. **Note:** Stop locations should have an error of no more than four meters when compared to the actual stop position.
 
-<!-- ## Stop details
+Optionally, additional parameters can be included such as:
+- **Stop Code (`stop_code`):** A short code that uniquely identifies a stop for passengers.
+- **Stop Name (`stop_name`):** The name of the location for a stop.
+- **Stop Description (`stop_desc`):** A description providing additional information about the stop.
+- **Zone ID (`zone_id`):** The fare zone for a stop.
+- **Stop Timezone (`stop_timezone`):** Timezone of the stop.
+- **Level ID (`level_id`):** Level of the location.
+- **Platform Code (`platform_code`):** Code that identifies the platform for a stop. For stations with multiple platforms.
+- **Parent Station (`parent_station`):** Defines the hierarchy of stops in `stops.txt`.
+- **Wheelchair Boarding (`wheelchair_boarding`):** A value (0, 1, or 2) that identifies whether wheelchair boardings are possible at the stop.
 
-- Name – the name of the stop
-- Other information – for any other details you may want to note down about the stop -->
+### Moving a stop
 
-## View all stops for feed
+To move a selected stop, **click and drag the stop to the new location**. Or, if already you know the latitude and longitude coordinates, you can copy these into the text fields. After moving the stop, click save to keep the changes.
```

Comment:
"If you already know"...

Reply:
What do you mean by this?

Reply:
"if already you know" => "if you already know"